go/oasis-node/cmd/registry/runtime/runtime.go

```diff
@@ -436,6 +465,56 @@ func runtimeFromFlags() (*registry.Runtime, error) { // nolint: gocyclo
 		return nil, fmt.Errorf("invalid runtime admission policy")
 	}
 
+	// Constraints.
+	// TODO: Hack needed because viper's Sub doesn't work correctly.
```

Comment:
are they tracking this issue?

Reply:
Yeah there's a bunch of them. Also they are rewriting viper.

Reply:
lol ok gl to them